### DISCUSSION TITLE
client no sound in lxde

### DISCUSSION THREAD

**Discussion by brcisna** [ltsp/ltsp#263] [Q&A]
Hello,

After many updates over the last couple weeks, the client machines will not get sound if logging into an LXDE desktop. If the same clients log out of LXDE desktop,and log into a gnome desktop the speaker icon is showing on menubar as expected and sound works. At this point a logout and then log into LXDE desktop the speaker icon is showing and sound works.
After a client reboot,this process has to be re-created.
It is an easy fix now that i stumbled onto a fix,but this didn't happen until about a week or two ago.

Thank You

**Comment by alkisg** [comment]
Hi, this problem doesn't sound related to LTSP, so you'll get more attention if you direct it to the LXDE community.
